### PR TITLE
Author list component - with comma separated and truncated list helpers

### PR DESCRIPTION
**Pull request by davidpauljunior**
The read me describes what the author-list component does.

I made the comma separated and truncated lists available as mixins and utils for usage elsewhere too.

### CODE REVIEW

File: packages/global-author-list/scss/10-settings/author-list.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// base
+$author-list--font-size: 1.6rem;
```
Comment (ricky-lau) [inline packages/global-author-list/scss/10-settings/author-list.scss]: var names should start with `global-`?

File: packages/global-author-list/scss/50-components/author-list.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.c-author-list {
+	@include u-list-comma-separated;
+	font-size: $author-list--font-size;
+	color: $author-list--color;
+
+	&--compact {
```
Comment (ricky-lau) [inline packages/global-author-list/scss/50-components/author-list.scss]: shoudnt be nesting elements https://github.com/springernature/frontend-playbook/blob/master/css/bem-css.md#bem-best-practices

Comment (davidpauljunior) [reply packages/global-author-list/scss/50-components/author-list.scss]: ha twice in a row

**Comment by davidpauljunior** [comment]
@ricky-lau pushed fixes for those

File: packages/global-context/scss/30-mixins/lists.scss [hunk 3]
```diff
@@ -13,9 +13,27 @@
...
+
+@mixin u-list-comma-separated {
+	@include u-list-inline();
+
+	> li:not(:first-child):not(:last-child)::before {
+		content: ', ';
+	}
+
+	> li:last-child::before {
+		content: ' & ';
```
Comment (ricky-lau) [inline packages/global-context/scss/30-mixins/lists.scss]: I was trying to find out if there was an advantage to making the value of content unicode but cant find anything so i leave it up to you lol

Comment (davidpauljunior) [reply packages/global-context/scss/30-mixins/lists.scss]: 🤷‍♂

**Pull request merged**